e2e-tests/content-strings/pages/insurance/declarations/anti-bribery-code-of-conduct/index.js

```diff
@@ -0,0 +1,23 @@
+import { SHARED } from '../shared';
+import { LINKS } from '../../../../links';
+
+export const ANTI_BRIBERY_CODE_OF_CONDUCT = {
+  ...SHARED,
+  PAGE_TITLE:
+    'Do you have in place a code of conduct and written procedures of the type contemplated by Section(2) of the Bribery Act to discourage and prevent corrupt activity?',
+  VERSIONS: [
+    {
+      VERSION: 2,
```

Comment:
Should this be version 1, since the content has just moved?

Reply:
Same with all other declarations?

Reply:
Will change to version 1